### PR TITLE
User can't be able to add new exercise if their email is not verified

### PR DESCRIPTION
**Pull request by Harshit933**
# Added screen which redirects to the user form whenever the user tries to add an exercise when their email is not verified.

-
-
-

Related Issues (if applicable)

-

## Please check that the PR fulfills these requirements

- [ ] Set a 100 character limit in your editor/IDE to avoid white space diffs in the PR
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Added yourself to AUTHORS.md

### CODE REVIEW

**Comment by Harshit933** [comment]
fixes #258

**Comment by rolandgeider** [comment]
Great, will try to review this later today

File: lib/screens/add_exercise_screen.dart [hunk 1]
```diff
@@ -66,56 +75,115 @@ class _AddExerciseScreenState extends State<AddExerciseScreen> {
...
+    return Scaffold(
+      appBar: EmptyAppBar('Verify Email'),
+      body: Container(
+        padding: EdgeInsets.all(25),
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Text(
+                'Your profile needs to be verified in order to add any exercise',
```
Comment (rolandgeider) [inline lib/screens/add_exercise_screen.dart:149]: Can you mark the text as translatable? You just need to add an entry to app_en.arb and then you can use it with `AppLocalizations.of(context).foo` (sometimes the IDE doesn't pick this up right away and complains that foo doesn't exists, but the app builds just fine).

I would also suggest that we use the same text as in the web application: "You can only contribute exercises if your account is older than {{days}} days and have verified your email". We can just store the number of days in consts.dart with 14, at the moment this is not configurable.

File: lib/screens/add_exercise_screen.dart [hunk 2]
```diff
@@ -66,56 +75,115 @@ class _AddExerciseScreenState extends State<AddExerciseScreen> {
...
+                    arguments: FormScreenArguments(
+                      AppLocalizations.of(context).userProfile,
+                      UserProfileForm(_user!),
+                    ),
+                  );
+                },
+                child: Container(
+                  padding: EdgeInsets.all(15),
+                  child: Text(
+                    'Profile Screen',
```
Comment (rolandgeider) [inline lib/screens/add_exercise_screen.dart:174]: Same here, just use `AppLocalizations.of(context).userProfile` (and something like a TextButton?)

**Comment by Harshit933** [comment]
Got it!

**Comment by Harshit933** [comment]
@rolandgeider done all the fixes!

**Comment by rolandgeider** [comment]
Merged, thanks

**Pull request merged**